src/CSET/operators/collapse.py

```diff
@@ -178,8 +221,14 @@ def collapse_by_hour_of_day(
         collapsed_cube = cube.aggregated_by("hour", getattr(iris.analysis, method))
     # Remove unnecessary time coordinates.
     collapsed_cube.remove_coord("time")
-    collapsed_cube.remove_coord("forecast_reference_time")
     collapsed_cube.remove_coord("forecast_period")
+    # Remove forecast_reference_time if a single case, as collapse_by_lead_time
+    # will have effectively done this if multi_case is True.
+    if not multi_case:
+        collapsed_cube.remove_coord("forecast_reference_time")
+    # Promote "hour" to dim_coord if monotonic.
```

Comment:
```suggestion

    # Promote "hour" to dim_coord if monotonic.
```
Space out for readability.